docs/source/intro/configure.rst

```diff
@@ -18,9 +18,9 @@ project template:
 
 .. code-block:: bash
 
-    $ django-admin startproject mysite --template https://github.com/mongodb-labs/django-mongodb-project/archive/refs/heads/5.1.x.zip
+    $ django-admin startproject mysite --template https://github.com/mongodb-labs/django-mongodb-project/archive/refs/heads/5.2.x.zip
```

Comment:
@aclark4life can you make the new ref for django-mongodb-project?

Reply:
Done!